integration/tests/epochs/suite.go

```diff
@@ -632,3 +680,33 @@ func (s *Suite) assertNetworkHealthyAfterVNChange(ctx context.Context, _ templat
 	// head should now be at-least 20 blocks higher from when we started
 	require.True(s.T(), header.Height-bootstrapHead.Height >= 20, fmt.Sprintf("expected head.Height %d to be higher than head from the snapshot the node was bootstraped with bootstrapHead.Height %d.", header.Height, bootstrapHead.Height))
 }
+
+// assertNetworkHealthyAfterLNChange after an collection node is removed or added to the network
+// this func can be used to perform sanity.
+// 1. Submit transaction to network that will target the newly staked LN by making sure the reference block ID
+// is after the first epoch starts.
+// 2. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
+func (s *Suite) assertNetworkHealthyAfterLNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo) {
+	bootstrapHead, err := rootSnapshot.Head()
+	require.NoError(s.T(), err)
+
+	header, err := s.client.GetLatestSealedBlockHeader(ctx)
+	require.NoError(s.T(), err)
+
+	// head should now be at-least 20 blocks higher from when we started
+	require.Truef(s.T(), header.Height-bootstrapHead.Height >= 20, "expected head.Height %d to be higher than head from the snapshot the node was bootstraped with bootstrapHead.Height %d.", header.Height, bootstrapHead.Height)
+
+	fullAccountKey := sdk.NewAccountKey().
+		SetPublicKey(unittest.PrivateKeyFixture(crypto.ECDSAP256, crypto.KeyGenSeedMinLenECDSAP256).PublicKey()).
+		SetHashAlgo(sdkcrypto.SHA2_256).
+		SetWeight(sdk.AccountKeyWeightThreshold)
+
+	// submit transaction to create account
+	_, err = s.createAccount(
```

Comment:
It would be great to add some notes about requirements this statement has, which are enforced elsewhere in the code. Eg:
* at this point we have entered epoch 1, where the joining LN is the only LN in the network, this is enforced in `runTestEpochJoinAndLeave`
* to validate the LN successfully processed the transaction, it must be sealed - this is checked within `createAccount`

Reply:
https://github.com/onflow/flow-go/pull/1819/commits/4436b1ea50cc954e3019dcdabb4bb3727dcf9263